packages/ui/src/app/pages/Council/PastCouncils/PastCouncil.tsx

```diff
@@ -19,12 +19,12 @@ import { usePastCouncil } from '@/council/hooks/usePastCouncil'
 export const PastCouncil = () => {
   const history = useHistory()
 
-  const { id } = useParams<{ id: string }>()
-  const { isLoading, council } = usePastCouncil(id)
+  const { cycleId } = useParams<{ cycleId: string }>()
+  const { isLoading, council } = usePastCouncil(+cycleId)
 
   useEffect(() => {
     if (!isLoading && !council) {
-      history.replace('/404')
+      history.replace('/council/past-councils')
```

Comment:
Test seems to be failing because of this change:
<img width="506" alt="image" src="https://user-images.githubusercontent.com/9252017/212420152-9733fd3d-5a6e-4285-be40-7c93008edf72.png">

Reply:
Thanks! Reverted as per https://github.com/Joystream/pioneer/pull/3959#issuecomment-1382407704